# HG changeset patch
# User dsamersoff
# Date 1304432695 -14400
# Node ID 250642c729b42043ea99e5e4e27d153b0c766da0
# Parent  405c634f4aaa2e91cc87598c60a4ece6e345e5b1
7041156: gcc 4.6 doesn't recognise -export-dynamic option
Summary: Since -export-dynamic is a linker option, the correct way to pass it is -Xlinker -export-dynamic
Reviewed-by: dsamersoff, dholmes
Contributed-by: omajid@redhat.com

diff -r 405c634f4aaa -r 250642c729b4 make/linux/makefiles/gcc.make
--- openjdk/hotspot/make/linux/makefiles/gcc.make	Mon May 02 14:53:49 2011 -0700
+++ openjdk/hotspot/make/linux/makefiles/gcc.make	Tue May 03 18:24:55 2011 +0400
@@ -205,7 +205,7 @@
 SHARED_FLAG = -shared
 
 # Keep symbols even they are not used
-AOUT_FLAGS += -export-dynamic
+AOUT_FLAGS += -Xlinker -export-dynamic
 
 #------------------------------------------------------------------------
 # Debug flags
